src/js/controllers/ctrl-collaboration.js

```diff
@@ -268,6 +268,11 @@ app.controller('CollaborationController', function(
 		VISIBLE: ACCESS.VISIBLE,
 		FULL: ACCESS.FULL
 	}
+	$scope.accessLevels = [
```

Comment:
This will have to change with the way it's being used in the template. Since the `model` refers to a value that is either `1` or `30`, this would probably make more sense as an object with keys `1` and `30` as opposed to an array.

```suggestion
	$scope.accessLevels = {}
	$scope.accessLevels[ACCESS.VISIBLE] = { value: ACCESS.VISIBLE, text: 'View Scores' }
	$scope.accessLevels[ACCESS.FULL] = { value: ACCESS.FULL, text: 'Full' }
```

See comments ucfopen/Materia#1231 for additional context.

Reply:
This would actually overcomplicate things - by adjusting the template to create `<option>`s explicitly using `ng-repeat`, the current array definition will work just fine.

The original issue was that newly added collaborators have a default access level which resolves to a mysterious and empty third option - that problem should probably be solved within this controller. Check the `_searchMatchClick` method where new collaborators are added and make sure their access levels make sense when compared to this array.